docs/llm-configuration.json

```diff
@@ -4,19 +4,20 @@
         "name": "openai",
         "title": "OpenAI",
         "modelList": [
-          "gpt-3.5-turbo",
-          "gpt-3.5-turbo-0301",
-          "gpt-4",
-          "gpt-4-1106-preview"
+          "gpt-4o-mini-2024-07-18",
+          "gpt-4o-mini",
+          "gpt-4o-2024-05-13",
+          "gpt-4-turbo-2024-04-09"
         ]
       }
     ],
     "llmIntegrationData": {
       "openai": {
-        "conversationModel": "gpt-4-1106-preview",
-        "pBGenerationModel": "gpt-4-1106-preview",
-        "isMultiConfigAvailable": false
+        "messagesLimit": 20,
```

Comment:
WE need to explain this value in the advance-usage section

Reply:
Done